spanner/spring-data/src/main/com/example/spanner/SpannerTemplateSample.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2019 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.spanner;
+
+
+import com.google.cloud.spanner.KeySet;
+import com.google.cloud.spanner.Statement;
+import java.util.List;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.cloud.gcp.data.spanner.core.SpannerTemplate;
+
+/**
+ * A quick start code for Spring Data Cloud Spanner. It demonstrates how to use SpannerTemplate to
+ * execute DML and SQL queries, save POJOs, and read entities.
+ */
+// [START spring_data_spanner_template_sample]
+@SpringBootApplication
+public class SpannerTemplateSample {
```

Comment:
See my comment on `SpannerRepositorySample`.

Reply:
as discussed separately, using the component annotation so that the 3 separate files for snippets are separated.